tests/testthat/test-stroke.R

```diff
@@ -197,14 +197,17 @@ test_that("a ring is recognized when from_edge is specified", {
 })
 
 test_that("flow mode does not break edges on a real dataset", {
-  edges <- sf::st_geometry(bucharest$streets)
+  edges <- sf::st_geometry(CRiSpData::bucharest_osm$streets)
 
   strokes <- rcoins::stroke(edges, flow_mode = TRUE)
 
   # find out which of the initial edges are contained in each of the strokes
-  # NOTE: edges included in self-intersecting strokes can be missed by the
-  # following command, if the test fails double check the input!
+  # NOTE: edges that form self-crossing strokes are missed by the "contains"
+  # relationship. However, they are catched by the "overlaps" relationship,
+  # which is why we run both predicate and merge results
   contains <- sf::st_contains(strokes, edges)
+  overlaps <- sf::st_overlaps(strokes, edges)
+  contains_and_overlaps <- mapply(c, contains, overlaps)
```

Comment:
Changes here due to: https://github.com/CityRiverSpaces/rcoins/pull/49#issuecomment-2765715837